bin/mocha

```diff
@@ -19,50 +20,52 @@ getOptions();
 process.argv.slice(2).forEach(arg => {
   const flag = arg.split('=')[0];
 
+  // every flag UNSHIFTED onto the `args` array is
+  // an flag for the `node` executable; every flag
+  // PUSHED onto the `args` array is a Mocha flag.
+
   switch (flag) {
+    // non-standard debug flags.
+    // any debug-related flag needs Mocha's --no-timeouts
+    // or Mocha will timeout while the debugger is paused.
     case '-d':
+    case 'debug':
+      // rename flag
       args.unshift('--debug');
       args.push('--no-timeouts');
       break;
-    case 'debug':
     case '--debug':
     case '--debug-brk':
     case '--inspect':
     case '--inspect-brk':
       args.unshift(arg);
       args.push('--no-timeouts');
       break;
+    // non-standard flag
     case '-gc':
-    case '--expose-gc':
+      // rename flag
       args.unshift('--expose-gc');
       break;
-    case '--gc-global':
```

Comment:
Why is this removed?

Reply:
this and the next one should be a special case for backwards compat.